backend/app/infrastructure/__init__.py

```diff
@@ -19,12 +21,29 @@ class OctantResource(Resource):
     def __init__(self, *args, **kwargs):
         Resource.__init__(self, *args, *kwargs)
 
+    @classmethod
```

Comment:
It's a very good idea to do it, but if we want to handle it once and for all, there are more:
- `address`
- `proposal_address` -> `project_address` if we change naming convention
- every address that comes from `subgraph` is lowercased, not checksummed, so we should decorate that as well
- I'm not sure how about calls to contracts directly

After changing it we should also remove all the invokes of `to_checksum_address` spread in the code. We can merge this one, but let's create a ticket to handle all the cases